packages/android_alarm_manager_plus/lib/android_alarm_manager_plus.dart

```diff
@@ -154,17 +162,15 @@ class AndroidAlarmManager {
     bool exact = false,
     bool wakeup = false,
     bool rescheduleOnReboot = false,
+    Map<String, dynamic> params = const {},
```

Comment:
Question: the new `params` is of type Map<String, dynamic> but then internally it gets parsed into a JSON. Wouldn't it cause an error passing a `dynamic` that is not serializable into JSON?

Reply:
You're right. I'll check that if the passing 'params' can be converted to json or not. If not it will throw an UnsupportedError .